tests/surveys/complex-survey/configuration.json

```diff
@@ -0,0 +1,89 @@
+{
+    "admin_name": "fastsurvey",
+    "survey_name": "complex-survey",
+    "title": "Complex Survey Test",
+    "description": "",
+    "start": 1000000000,
+    "end": 2000000000,
+    "mode": 1,
+    "fields": [
+        {
+            "type": "Email",
+            "title": "What is your email address?",
+            "description": "",
```

Comment:
We definitely have to think about a good distinction between `title` and `description` so that the generated UI looks good in all cases.

What are your thoughts on these two properties?

Reply:
**Solution:** Character limit on the `title`. Only render `description` when used.